docs/code_ref/io.rst

```diff
@@ -85,6 +87,14 @@ file
             -128.03072  , -128.03072  ]], dtype=float32)
    >>> input_asdf.close()  # doctest: +REMOTE_DATA
 
+When saving a Map to ASDF the specific Map source class is not saved, all maps are saved as a `.GenericMap`.
+This comes with some trade-offs.
+If you are using custom map sources defined outside of `sunpy`, and these sources are imported after asdf has been invoked (not imported, but used) for the first time then they will not be registered with the asdf converter.
+Also if the custom map subclass is not present (registered with `sunpy.map.Map`) upon loading of the map, it will be returned as a `.GenericMap`.
```

Comment:
```suggestion
Also if the custom map subclass is not registered with `sunpy.map.Map` upon loading of the map, it will be returned as a `.GenericMap`.
```